rtmr/rtmr_test.go

```diff
@@ -63,6 +63,26 @@ func TestExtendDigestRtmrOk(t *testing.T) {
 	}
 }
 
+func TestExtendDigestRtmrWithExistingEntriesOk(t *testing.T) {
```

Comment:
Please add a comment describing the error condition you ran into that you’re now testing the fix for. I’m not getting it. I’d also like you to confirm that the test fails for the real implementation on real hardware before the fix and succeeds after.